docroot/modules/custom/sitenow_articles/src/Entity/Article.php

```diff
@@ -30,6 +30,25 @@ public function buildCard(array &$build) {
 
     // Construct remaining byline.
     $build['#meta']['byline'] = $this->getByline($build);
+
+    // Add the published date.
+    $created = $this->get('created')->value;
+    $date = \Drupal::service('date.formatter')->format($created, 'medium');
+    $build['#subtitle'] = $date;
+
+    // Add view specific classes.
+    if (isset($this->view)) {
+      if ($this->view->id() === 'articles') {
+        $media_attributes = [
+          'card--layout-right',
+          'card--list',
+          'media--widescreen',
+          'media--small',
+          'borderless',
+        ];
+        $build['#attributes']->addClass($media_attributes);
+      }
+    }
```

Comment:
I don't think this is necessary. For each content type, we cover the "base" case of what the card should look like in the `getDefaultStyles()` method. Since the above view is not one where the user gets to pick styles, we should just be inheriting the default styles for consistencies sake.